src/libstd/process.rs

```diff
@@ -253,6 +253,10 @@ impl Command {
     /// Builder methods are provided to change these defaults and
     /// otherwise configure the process.
     ///
+    /// The `PATH` from the Command's environment will be searched
+    /// if the program contains only a single path component, as for
+    /// `execvp`.
```

Comment:
That isn't the case on Windows. It uses the child's `PATH` (badly: #37519) even if the program name contains multiple path components.

Windows and Linux have very different behaviour here.